### ISSUE TITLE
loss function choice

### ISSUE THREAD

**Issue by MacieJayDaaaaa** [YuYue525/MobileSAM-pytorch#9]
Hello,I trained a retained-model using your   customized_ mseloss , and my retrained model is around 0.6, while the official Mobilesam is around 1.4.  I think the official Mobilesam may use other loss function for training.Have you tried any other loss functions?

**Comment by YuYue525** [comment]
Same idea. You can try other loss functions. By the way, I tried and have some improvement.

**Comment by MacieJayDaaaaa** [comment]
> Same idea. You can try other loss functions. By the way, I tried and have some improvement.

If you could share your loss function, I would greatly appreciate it
